skimage/_shared/filters.py

```diff
@@ -4,14 +4,47 @@
 
 The unit tests remain under skimage/filters/tests/
 """
+import warnings
 from collections.abc import Iterable
+from functools import wraps
 
 import numpy as np
 from scipy import ndimage as ndi
 
 from .._shared.utils import _supported_float_type, convert_to_float
 
 
+def _deprecate_gaussian_positional_args(func):
+    """Help deprecate `output` in `gaussian` in favor of `out`.
+
+    We can't simply use deprecate_kwarg or remove_arg, as `output` can also be
+    passed as a positional argument. Instead, deprecate passing more than 1
+    positional argument and at the same time, refactor `output` to `out`.
+    """
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        if len(args) > 1:
+            warnings.warn(
+                "Passing more than 1 positional argument to `gaussian` is deprecated "
+                "and will stop working in version 0.25 or later. Please use keyword "
```

Comment:
```suggestion
                "and will stop working in version 0.25. Please use keyword "
```
Alternatively, "... will not work in versions 0.25 and later."

Unless I didn't get the meaning right, and you want to leave it open: We will decide later when it will stop working (either in version 0.25 or in a later version). But I'd rather communicate a definite version for when it stops working.